dotCMS/src/integration-test/java/com/dotcms/publishing/PublisherAPITest.java

```diff
@@ -319,4 +323,138 @@ private FileFilter getNoBundleXMLFileFilter() {
         return file -> !file.getAbsolutePath().endsWith("bundle.xml");
     }
 
+
+    @Test
+    public void test_addFilter(){
+        final Map<String,Object> filtersMap =
+                ImmutableMap.of("dependencies",true,"relationships",true,"excludeClasses","Host,Workflow");
+        final FilterDescriptor filterDescriptor =
+                new FilterDescriptor("filterTestAPI.yml","Filter Test Title",filtersMap,false,"Reviewer,dotcms.org.2789");
+
+        publisherAPI.addFilter(filterDescriptor);
+
+        final Map<String,FilterDescriptor> filterDescriptorMap = APILocator.getPublisherAPI().getFilterMap();
+        Logger.info(this,filterDescriptorMap.toString());
+        Assert.assertFalse(filterDescriptorMap.isEmpty());
+        Assert.assertTrue(filterDescriptorMap.containsKey(filterDescriptor.getKey()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertTrue' due to existing static import 'org.junit.Assert.assertTrue'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)